test/util/validator.test.js

```diff
@@ -0,0 +1,54 @@
+const { validateInputJson } = require('../../src/util/validator')
+
+test('validateInputJson', () => {
+  const goodRequest = {
+    amount: 20,
+    currencyCode: 'USD',
+    partnerId: 'yourPartnerId',
+    accessKey: 'yourAccessKey',
+    secretKey: 'yourSecretKey',
+    environment: 'sandbox',
+    endpoint: 'NA',
+  }
+
+  const stringFields = [ 'currencyCode', 'partnerId', 'accessKey', 'secretKey' ]
+  const enumFields = [ 'environment', 'endpoint' ]
+  const badStrings = [ 35, [], '' ]
+  const goodEnums = {
+    environment: [ 'sandbox', 'production' ],
+    endpoint: [ 'NA', 'EU', 'FE' ]
+  }
+
+  expect(() => validateInputJson(goodRequest)).not.toThrowError()
+  expect(() => validateInputJson(null)).toThrowError('JSON cannot be null')
+
+  expect(() => validateInputJson({ ...goodRequest, amount: '20' })).not.toThrowError()
+  expect(() => validateInputJson({ ...goodRequest, amount: 999999 })).not.toThrowError()
+  
+  expect(() => validateInputJson({ ...goodRequest, amount: 0 })).toThrowError()
```

Comment:
Make this into its own test. E.g., test('Invalid amount throws error', () => { //code here }

Make an array of invalid amounts and loop through them